config/config.default.php

```diff
@@ -53,6 +53,15 @@
         'Contact' => env('CONTACT_EMAIL', 'mailto:ticket@c3heaven.de'),
     ],
 
+    // Other contact points then ask the heaven
+    // You can set as many contact points as you like analog to email some
```

Comment:
I am missing some more structure in that description

Reply:
do you have any specific proposals?

Reply:
Maybe something like
```
    // Other ways to ask the heaven
    // Multiple contact options / links are possible, analogue to footer_items
```